sklearn/pipeline.py

```diff
@@ -609,12 +631,17 @@ def _transform_one(transformer, X, y, weight, **fit_params):
     return res * weight
 
 
-def _fit_transform_one(transformer, X, y, weight, **fit_params):
-    if hasattr(transformer, 'fit_transform'):
-        res = transformer.fit_transform(X, y, **fit_params)
-    else:
-        res = transformer.fit(X, y, **fit_params).transform(X)
-    # if we have a weight for this transformer, multiply output
+def _fit_transform_one(is_transform, clsname, message,
```

Comment:
took me a minute to figure out we have to add these to the front to not confuse them with fit_params...
Maybe this function can get a docstring also? ;)